src/coprocessor/dag/builder.rs

```diff
@@ -46,6 +46,12 @@ impl DAGBuilder {
                     let descriptor = ed.get_selection();
                     BatchSelectionExecutor::check_supported(&descriptor)?;
                 }
+                ExecType::TypeAggregation | ExecType::TypeStreamAgg
```

Comment:
should also consider `TypeAggregation`?

Reply:
Yes TypeAggregation is considered in case of TiDB not correctly use stream agg.